packages/cactus-plugin-ledger-connector-iroha2/src/test/typescript/integration/iroha-instructions-and-queries.test.ts

```diff
@@ -0,0 +1,591 @@
+/**
+ * Tests for executing Iroha instructions and queries through the cactus connector.
```

Comment:
@outSH Please make sure to include the Iroha version and the in the comments and the file name (it's not done consistently elsewhere in the code either so apologies for asking for this additional clarity).

Reply:
@petermetz Which file name? The test file name?

Reply:
@outSH Yeah. I posted a suggestion edit patch as a comment here: https://github.com/hyperledger/cactus/pull/2153#issuecomment-1299342415 to show what I mean (rename + also in the comment I said Iroha 2 instead of just Iroha